source/assets/css/components/_index.scss

```diff
@@ -0,0 +1,7 @@
+@include sl-breakpoint--medium {
```

Comment:
Can this go in the components/callouts file instead?

Reply:
Done (although this mostly doesn't have to do with callouts).